src/test/java/org/gridsuite/voltageinit/server/VoltageInitControllerTest.java

```diff
@@ -105,18 +102,19 @@ public class VoltageInitControllerTest {
     private final RestTemplateConfig restTemplateConfig = new RestTemplateConfig();
     private final ObjectMapper mapper = restTemplateConfig.objectMapper();
 
-    private Network network;
+    private Network network = EurostagTutorialExample1Factory.createWithMoreGenerators(new NetworkFactoryImpl());
```

Comment:
you should keep init the network in the setup to have a different instance at each test

Reply:
Yes done